### DISCUSSION TITLE
Crop control in Lightroom

### DISCUSSION THREAD

**Discussion by TGM12345** [rsjaffe/MIDI2LR#681] [Q&A]
Hi,

Is there a way to increase/decrease the size of the crop area using one of the knobs as is possible in Loupedeck? 

Thanks,

Tom

**Comment by PitToad** [comment]
'Crop All' crops the pic to whatever ratio you have set
If you assign a couple of other dials to 'Crop - Move Vert' and 'Crop - Move Horiz' you can then move the crop window around

![crop](https://user-images.githubusercontent.com/111119318/186263269-c07aa956-63c8-4541-b8d2-4054ab02eff5.jpg)

**Comment by TGM12345** [comment]
Hi,

Thank you.

This was not quite what I was asking.

Is there a way to shrink the crop size inside a knob while keeping the ratio the same?

Best,

Tom

**Comment by PitToad** [comment]
Once you've assigned 'Crop All' to a dial, you can shrink/enlarge the crop window from 100% to 0%  (although 0 is just really tiny, not non-existant)
You then have to assign a button to 'Return' or use the return key to apply the crop. 
Did you want it to do something different to that?

![crop1](https://user-images.githubusercontent.com/111119318/186335701-451a7692-d6ed-49e4-ab45-2fdc9c81d248.jpg)

![crop2](https://user-images.githubusercontent.com/111119318/186335708-3b21f757-317d-4e7e-a44e-e8104149e193.jpg)

![crop3](https://user-images.githubusercontent.com/111119318/186335713-647a4401-9390-4364-b14b-903380782115.jpg)

![crop4](https://user-images.githubusercontent.com/111119318/186335716-a0abbdf6-56fe-4925-8cee-8d46338b4243.jpg)

**Comment by TGM12345** [comment]
Yes! Thank you!

------- Original Message -------

**Comment by TGM12345** [comment]
Hi,

Thanks again for your help.

Is there a command to move the crop left/right? The "Crop - move horizontally/vertically" or "crop - move left/right" does not see to do that.

Thank you!

Tom

------- Original Message -------

**Comment by PitToad** [comment]
Left/Right is 'Crop - Move Horizontally'
So, you set the crop window, then move the crop window around left/right and, if needed,  up/down with Crop - Move Vertically
The options are toward the bottom of the crop functions (see pic below)

Personally, I crop with a mouse - having 3 dials reserved for cropping is too many for my setup

![186263269-c07aa956-63c8-4541-b8d2-4054ab02eff5](https://user-images.githubusercontent.com/111119318/186913801-0684d8bc-fe29-4201-98ae-0d7da51cefbd.jpg)

**Comment by TGM12345** [comment]
Thank you!

------- Original Message -------
